src/arrays.js

```diff
@@ -1,61 +1,72 @@
 const getNthElement = (index, array) => {
-  // your code here
+if (index > array.length -1) {
+  return array[index % array.length]
+} else { return array[index]}
 };
 
 const arrayToCSVString = (array) => {
-  // your code here
+  return array.toString()
 };
 
 const csvStringToArray = (string) => {
-  // your code here
+  return string.split(',')
 };
 
 const addToArray = (element, array) => {
-  // your code here
+   array.push(element)
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+  return newArray2 = array.concat(element);
```

Comment:
You don't need to return assignment here (and you shouldn't ever - only ever return a value (but never `undefined`).

```js
return array.concat(element)
```

[Array.prototype.concat() on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat) - "This method does not change the existing arrays, but instead returns a new array."